README.md

```diff
@@ -72,31 +72,31 @@ The `WitnetRequestBoard` implements the following functionality:
   - _Returns_:
     - *_id*: the unique identifier of the data request.
 
-- **`readDataRequest(uint256 _id)`**:
+- **`readRequestBytecode(uint256 _id)`**:
   - _Description_:
     - Retrieves the RADON script bytecode of a previously posted Witnet data request.
   - _Inputs_:
     - `_id`: the unique identifier of a previously posted Witnet data request.
   - _Returns_:
     - The Witnet RADON script bytecode, as `bytes`.
 
-- **`readResult(uint256 _id)`**:
+- **`readResponseWitnetResult(uint256 _id)`**:
```

Comment:
There already exists a `readResponse` that returns the whole `WitnetData.Response` struct. I agree though with keeping `readResult` instead of the verbose `readResponseResult`.

Reply:
Ok,`readResponseResult` be it! This is opaque to `UsingWitnet` anyway.